src/torchmetrics/multimodal/clip_iqa.py

```diff
@@ -259,3 +259,18 @@ def plot(self, val: Union[Tensor, Sequence[Tensor], None] = None, ax: Optional[_
 
         """
         return self._plot(val, ax)
+
+
+if TYPE_CHECKING:
+    from typing import cast
+
+    images = cast(Any, None)
```

Comment:
I guess I can safely delete this line, but I'm outside right now so I'll get it done when I get home tonight.

Reply:
```suggestion
```
we can do it here with suggestions and commit :)